### DISCUSSION TITLE
hw7-function-partial-order problem

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#10] [Q&A] [answered]
老师我又来了：

![image](https://user-images.githubusercontent.com/77151639/123227989-b0a4a680-d507-11eb-9c25-f473f0a1cee2.png)

这里所说的 $f(a)$ 不是集合，难道 $f(a)$ 指的不是函数 $f$ 的陪域吗？
如果按照图中助教所写的 $f(a)$ 不是集合，那么该如何表示 $f$ 的陪域呢，直接写 $cod(f)$ 来表示 $f$ 的陪域吗？

**Comment by hengxin** [comment, marked as answer]
@Martin7-1  (8) 不需要, 从 (7) 到 (9) 毫无违和感。如果要说 f 的陪域, 用 cod(f)，或者定义域 (设为 A) 的像 f(A)。

**Reply by Martin7-1** [reply]
明白了。谢谢老师。
